packages/calling-stateful-client/src/CallAgentDeclarative.test.ts

```diff
@@ -80,10 +77,9 @@ class MockCallAgent implements CallAgent {
     return call;
   }
   join(groupLocator: GroupLocator, options?: JoinCallOptions): Call;
-  join(groupChatCallLocator: GroupChatCallLocator, options?: JoinCallOptions): Call;
-  join(meetingLocator: MeetingLocator, options?: JoinCallOptions): Call;
+  join(TeamsMeetingLinkLocator: TeamsMeetingLinkLocator, options?: JoinCallOptions): Call;
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  join(meetingLocator: any, options?: any): Call {
+  join(TeamsMeetingLinkLocator: any, options?: any): Call {
```

Comment:
Upper case? Lower case? (For first 'T'), this one with any is just implementation, we shouldn't expose it or we should keep it as meetingLocator